### PR TITLE
delete criteria filter

### PR DESCRIPTION
**Pull request by basseche**
to test you must use commons-ui version :

https://github.com/gridsuite/commons-ui/pull/648

### CODE REVIEW

File: src/components/directory-content.tsx [hunk 1]
```diff
@@ -10,7 +10,6 @@ import { useDispatch, useSelector } from 'react-redux';
 import { FormattedMessage, useIntl } from 'react-intl';
 import { Box, Button, CircularProgress, Grid, SxProps, Theme } from '@mui/material';
 import {
-    CriteriaBasedFilterEditionDialog,
```
Comment (ghazwarhili) [inline src/components/directory-content.tsx:13]: CriteriaBasedFilterEditionDialog is not used anymore 
to be removed

Comment (basseche) [reply src/components/directory-content.tsx:13]: I don't understand ! where i should remove that from ?

**Review by Mathieu-Deharbe — COMMENTED** [review]
`CRITERIA_BASED: { id: 'CRITERIA', label: 'filter.criteriaBased' },` from elementTypes may probably be removed.

Edit : Done.

File: src/components/directory-content.tsx (hunk 1 above)
Comment (ghazwarhili) [reply src/components/directory-content.tsx:13]: in commons-ui

Comment (basseche) [reply src/components/directory-content.tsx:13]: I think it is already the case

**Review by ghazwarhili — APPROVED** [review]

**Review by Mathieu-Deharbe — APPROVED** [review]

**Pull request merged**